prac_09/taxi.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+Car class
+"""
+from prac_09.car import Car
```

Comment:
If the car.py file is in the same folder as where the program py, you can just write car